game_template/lib/src/crashlytics/crashlytics.dart

```diff
@@ -40,14 +40,14 @@ Future<void> guardWithCrashlytics(
       crashlytics?.log(message);
 
       if (record.level >= Level.SEVERE) {
-        crashlytics?.recordError(message, filterStackTrace(StackTrace.current));
+        crashlytics?.recordError(message, filterStackTrace(StackTrace.current), fatal: true);
```

Comment:
Bets this line is too long for the formatter.

Reply:
Thanks you. I was wondering what the issue was. Dart format done.